py/server/deephaven/table_listener.py

```diff
@@ -265,6 +285,13 @@ def __init__(self, t: Table, listener: Union[Callable[[TableUpdate, bool], None]
                 the listener is safe, it is not recommended because reading or operating on the result tables of those
                 operations may not be safe. It is best to perform the operations on the dependent tables beforehand,
                 and then add the result tables as dependencies to the listener so that they can be safely read in it.
+            on_error (Callable[[Exception], None]): a callback function to be invoked when an error occurs during the
+                listener's execution. It should only be set when the listener is a function, not when it is an instance
+                of TableListener. Defaults to None. When None, a default callback function will be provided that simply
+                prints out the received exception. If the callback function itself raises an exception, the new exception
+                will be logged in the Deephaven server log and will not be further processed by the server.
```

Comment:
on_error (Callable[[Exception], None]): a callback function to be invoked when an error occurs during the
                listener's execution. It should only be set when the listener is a function, not when it is an instance
                of TableListener. When the listener is a TableListener, TableListener.on_error will be used.  Defaults to None. When None, a default callback function will be provided that simply
                prints out the received exception. If the callback function itself raises an exception, the new exception
                will be logged in the Deephaven server log and will not be further processed by the server.

... similar change in other `on_error` docs